src/current/v23.1/migration-overview.md

```diff
@@ -115,7 +115,7 @@ Determine the size of the target CockroachDB cluster. To do this, consider your
 
 Use this information to size the CockroachDB cluster you will create. If you are migrating to a CockroachDB {{ site.data.products.cloud }} cluster, see [Plan Your Cluster]({% link cockroachcloud/plan-your-cluster.md %}) for details:
```

Comment:
I think we should only link there and not discuss Cloud further in this self-hosted page. I know this content was already there, but I think it's incorrectly placed.

Reply:
Flagging to discuss this section with Mark and how it can work with his new content on Standard > Plan Your Cluster.

Reply:
My point is that we have that info already in the link target. We don't need the link if we restate the facts here, we don't need to restate the facts if we are sending them to the link.